include/realtime_memory/utilities.h

```diff
@@ -0,0 +1,95 @@
+//==============================================================================
+// Copyright (c) 2019-2022 CradleApps, LLC - All Rights Reserved
```

Comment:
```suggestion
// Copyright (c) 2019-2023 CradleApps, LLC - All Rights Reserved
```
but could do as a separate pass.